tools/c7n_azure/c7n_azure/function_package.py

```diff
@@ -78,7 +81,15 @@ def _add_host_config(self):
             }
         self.pkg.add_contents(dest='host.json', contents=json.dumps(config))
 
-    def get_function_config(self, policy):
+    def _add_queue_binding_extensions(self):
+        bindings_dir_path = os.path.abspath(
+            os.path.join(os.path.join(__file__, os.pardir), 'function_binding_resources'))
+        bin_path = os.path.join(bindings_dir_path, 'bin')
+
+        self.pkg.add_directory(bin_path)
+        self.pkg.add_file(os.path.join(bindings_dir_path, 'extensions.csproj'))
```

Comment:
nit: do we need this csproj file in the package?

Reply:
It gets sent up with the func core tools also, but I can remove it and see if you think it's important.

Reply:
nope, let's do whatever core tools is doing.